openshift/bddstack.pod.json

```diff
@@ -136,19 +132,32 @@
             ],
             "resources": {
               "limits": {
-                "cpu": "1.5",
-                "memory": "3Gi"
+                "cpu": "0",
+                "memory": "0"
               },
               "requests": {
-                "cpu": "1",
-                "memory": "1Gi"
+                "cpu": "0",
+                "memory": "0"
+              }
+            },
+            "volumeMounts": [
```

Comment:
Selenium has some issue running in containers, so it needs host memory shared to work 🤷‍♂